package.json

```diff
@@ -23,19 +23,23 @@
     "@types/express": "^4.17.8",
     "@types/helmet": "^0.0.48",
     "@types/http-status-codes": "^1.2.0",
+    "@types/jsonwebtoken": "^8.5.2",
```

Comment:
Anything related to @types should be in devDependencies. Why? The reason is because it helps with the development but does not play any part in production.
`yarn add @types/jsonwebtoken -D` or `npm install @types/jsonwebtoken --save-dev` should work

Reply:
oh really, I didn't know about this before, great tips!! Indeed types is needed only in dev env

Reply:
This is good practice when it comes to webpack and I really want to introduce webpack since it's a really good tool. 
FYI webpack basically includes all the dependencies (not dev dependencies), transpiles and minimizes it so when it gets uploaded to the cloud, the size is much smaller.